### ISSUE TITLE
[PS3] MENU SHADER PIPELINE Settings Causes Console Freeze

### ISSUE THREAD

**Issue by STLcards** [libretro/RetroArch#6693]
Port: PlayStation 3
Release Version: RetroArch v1.7.2 
Issues: FREEZE from changing MENU SHADER PIPELINE.
---- In the XMB View, when changing options in the "MENU Shader Pipeline" Settings, the PS3 will freeze. You never see the Simple Snow or some of the ones. You are able to scroll a few depending on which way you go (left or right).

**Comment by twinaphex** [comment]
Yeah, those pipelines are simply not implemented for Cg. I wouldn't advise trying to go beyond the Simple Ribbon.

**Comment by STLcards** [comment]
The Ribbon and Simplified Ribbon both seems to work fine, its just anything beyond that. Looking at those other options on other ports did not think there was much hope for the PS3.  

My main concern is the console freezes , Would it be possible to omit those options from the port?
